datafusion/sql/src/planner.rs

```diff
@@ -530,18 +530,18 @@ pub(crate) fn idents_to_table_reference(
     match taker.0.len() {
         1 => {
             let table = taker.take(enable_normalization);
-            Ok(OwnedTableReference::bare(&table))
+            Ok(OwnedTableReference::bare(table))
```

Comment:
somehow it seems the compiler can avoid copying / allocating by getting this as an owned item